.stylelintrc.js

```diff
@@ -17,7 +17,6 @@ module.exports = {
         "color-function-notation": null, // DISABLE: Expected modern color-function notation
         "declaration-block-no-redundant-longhand-properties": null, // DISABLE Expected shorthand property "flex-flow"
         "media-feature-range-notation": "prefix",
-        "selector-not-notation": "simple", //DISABLE Expected complex :not() pseudo-class notation
```

Comment:
I need a complex selector for a case that do not seems possible with simple selectors.

I've removed this rule, but it requires us to remove usages of simple selectors so I've had to fix a few rules using the automated `--fix` command.

See https://stylelint.io/user-guide/rules/selector-not-notation/.

Reply:
Doesn't seem to be an issue. All officially supported browsers support selector lists.
https://caniuse.com/css-not-sel-list